open_spiel/scripts/install.sh

```diff
@@ -105,9 +105,10 @@ fi
 # This Github repository contains the raw code from the ACPC server
 # http://www.computerpokercompetition.org/downloads/code/competition_server/project_acpc_server_v1.0.42.tar.bz2
 # with the code compiled as C++ within a namespace.
-DIR="open_spiel/games/universal_poker/acpc"
+DIR="open_spiel/games/universal_poker"
```

Comment:
Why are you changing this? (I'm fine changing it, but what's the reason?)

Reply:
to be able to reuse `DIR` in git clones